kitsune/sumo/static/sumo/js/questions.js

```diff
@@ -104,6 +108,12 @@
     hideDetails($questionForm, aaq);
   }
 
+  function initEditQuestion() {
```

Comment:
nit: Can we maybe combine this with `initNewQuestion` based on whether it's a new or an existing question since they are sharing some code?